src/analysis/souffle/taint.dl

```diff
@@ -123,13 +123,24 @@ isTag(tag) :- mayHaveTag(_, tag).
 isTag(tag) :- ownsTag(_, tag).
 isTag(tag) :- saysDowngrades(_, _, tag).
 
+// A pair of (base, member) is in this set if the base access path is a non-trivial subpath of the
```

Comment:
Wait for #59  to be merged and move this code into `access_path.dl`?

Reply:
Sure, can do.

Reply:
#59 is merged now.

Reply:
Acknowledged.